scripts/qa_crowbarsetup.sh

```diff
@@ -4147,7 +4147,7 @@ function onadmin_addupdaterepo
 function zypper_patch
 {
     wait_for 30 3 ' zypper --non-interactive --gpg-auto-import-keys --no-gpg-checks ref ; [[ $? != 4 ]] ' "successful zypper run" "exit 9"
-    wait_for 30 3 ' zypper --non-interactive patch ; ret=$?; if [ $ret == 103 ]; then zypper --non-interactive patch ; ret=$?; fi; [[ $ret != 4 ]] ' "successful zypper run" "exit 9"
+    wait_for 30 3 ' zypper --non-interactive patch --with-interactive ; ret=$?; if [ $ret == 103 ]; then zypper --non-interactive patch --with-interactive ; ret=$?; fi; [[ $ret != 4 ]] ' "successful zypper run" "exit 9"
```

Comment:
so I heard, this is to avoid trouble with our sanity-checks in the upgrade?
a) I would have liked the commit message to say so and
b) it would be nice if the sanity-check could ignore pending kernel updates instead, because those will not matter anyway